2021-11-09 15:31:55,813 INFO    MainThread:25208 [wandb_setup.py:_flush():69] setting env: {}
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_153155-3g6g9j37\logs\debug.log
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_153155-3g6g9j37\logs\debug-internal.log
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_init.py:init():375] calling init triggers
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-11-09 15:31:55,814 INFO    MainThread:25208 [wandb_init.py:init():424] starting backend
2021-11-09 15:31:55,814 INFO    MainThread:25208 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-11-09 15:31:55,819 INFO    MainThread:25208 [backend.py:ensure_launched():135] starting backend process...
2021-11-09 15:31:55,917 INFO    MainThread:25208 [backend.py:ensure_launched():139] started backend process with pid: 17464
2021-11-09 15:31:55,917 INFO    MainThread:25208 [wandb_init.py:init():429] backend started and connected
2021-11-09 15:31:55,918 INFO    MainThread:25208 [wandb_init.py:init():477] updated telemetry
2021-11-09 15:31:55,919 INFO    MainThread:25208 [wandb_init.py:init():500] communicating current version
2021-11-09 15:31:57,329 INFO    MainThread:25208 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-09 15:31:57,329 INFO    MainThread:25208 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-11-09 15:31:57,610 INFO    MainThread:25208 [wandb_init.py:init():540] starting run threads in backend
2021-11-09 15:32:02,506 INFO    MainThread:25208 [wandb_run.py:_console_start():1601] atexit reg
2021-11-09 15:32:02,508 INFO    MainThread:25208 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-11-09 15:32:02,508 INFO    MainThread:25208 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-11-09 15:32:02,511 INFO    MainThread:25208 [wandb_run.py:_redirect():1536] Redirects installed.
2021-11-09 15:32:02,511 INFO    MainThread:25208 [wandb_init.py:init():565] run started, returning control to user process
2021-11-09 15:32:02,511 INFO    MainThread:25208 [wandb_config.py:__setitem__():141] config set data = {'name': 'bike'} - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001AD57F4D8B0>>
2021-11-09 15:32:02,511 INFO    MainThread:25208 [wandb_run.py:_config_callback():843] config_cb data {'name': 'bike'} None
2021-11-09 15:33:20,333 INFO    MainThread:25208 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-11-09 15:33:20,640 INFO    MainThread:25208 [wandb_run.py:_restore():1543] restore
2021-11-09 15:33:21,352 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 945
}

2021-11-09 15:33:21,655 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 945
}

2021-11-09 15:33:22,328 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 16092
}

2021-11-09 15:33:22,443 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 16119
}

2021-11-09 15:33:22,568 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 16119
}

2021-11-09 15:33:22,676 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:22,783 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:22,895 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:23,005 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:23,116 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:23,306 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:23,416 INFO    MainThread:25208 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16119
  total_bytes: 16119
}

2021-11-09 15:33:25,052 INFO    MainThread:25208 [wandb_run.py:_show_files():1915] logging synced files
